data/repos.yml

```diff
@@ -468,20 +588,44 @@
   team: "#data-products"
   type: Data science
 
+# Guessing, based on existing entry for `govuk-feedback-pipeline`?
+- repo_name: govuk-feedback-analysis
+  team: "#data-products"
+  type: Data science
+
 - repo_name: govuk-feedback-pipeline
   private_repo: true
   team: "#data-products"
   type: Data science
 
+# ditto
+- repo_name: govuk-feedback-processing
+  team: "#data-products"
+  type: Data science
+
+# ditto
+- repo_name: govuk-feedback-visualisation
+  team: "#data-products"
+  type: Data science
+
 - repo_name: govuk-feedback-spam-classifier
   private_repo: true
   team: "#data-products"
   type: Data science
 
+# Propose deleting or archiving. Looks like a temporary repo written for GIFT week, where the main code lives in Simon's account.
+- repo_name: govuk-formats
+
+# Propose the User Experience and Measurement team add this in a PR
+- repo_name: govuk-google-analytics
```

Comment:
@gclssvglx ?

Reply:
Another one, if you have time please @ChrisBAshton https://github.com/alphagov/govuk-developer-docs/pull/3916